### DISCUSSION TITLE
What do all these question marks mean in the file explorer?

### DISCUSSION THREAD

**Discussion by plasmoduck** [LazyVim/LazyVim#4363] [General]
What do all these question marks mean in the file explorer? Also why are some names green and pink (different colors) and some have a dot and some a question mark?

![2024-08-26_23-47-38](https://github.com/user-attachments/assets/6eaacea6-0fb4-495a-b734-3a392bdbe56f)

**Comment by cerebrl** [comment]
You're likely missing the icons in your chosen icon font. The pink color is probably an indication of git status, probably a changed/uncommitted files. The icons on the right column, where all the question marks are, is the git status icon set. You can try a different icon font in your terminal, or change the glyph in your plugin's configuration to map to something your icon font provides.

**Reply by plasmoduck** [reply]
Thanks. I am Using FantasqueSans Mono Nerd Font and JetBrains Mono Nerd Font so it should display correctly. What file do I edit to change the icons for Lazyvim?

**Reply by cerebrl** [reply]
If this is Neotree I'm seeing in the screenshot, it's probably using an older version of Nerd Font, and you're using the latest version. I think some of the character codes have changed. I don't use Neotree anymore, but here's a commit in my old config that fixed the issue for me at the time: https://github.com/cerebrl/nvim/commit/d191d14c04af27f323a2ed7e0ecbc1ce78a03c23#diff-2478f550ee2f7c5c2397978ddcdf072228138c4d86b3d9eb4f838155c847251fR32

**Reply by plasmoduck** [reply]
Thanks, I fixed all the green by removing ~/.config/.git I forgot I had my config dir as an old git repo. Looks much better now.

![2024-08-27_01-22-43](https://github.com/user-attachments/assets/d4f380db-ba8c-4e07-83d4-e2828a9e6577)
